### ISSUE TITLE
Encoding issues for a number of formats in Python 2.x series

### ISSUE THREAD

**Issue by benjwadams** [ioos/compliance-checker#254]
Trying to output tests to html results in a `TypeError`s in Python 2.x series for ACDD, a `UnicodeDecodeError` for cf output to HTML.

**Comment by daf** [comment]
What file(s)?  Any?

**Comment by benjwadams** [comment]
Only really got around to testing `compliance_checker/tests/data/2dim-grid.nc`, but prelimnary tests show issues with other files as well.

```
compliance-checker --test acdd -f html -- compliance_checker/tests/data/2dim-grid.nc
Traceback (most recent call last):
  File "/home/badams_local/.virtualenvs/cchecker/bin/compliance-checker", line 9, in <module>
    load_entry_point('compliance-checker==2.1.0', 'console_scripts', 'compliance-checker')()
  File "/home/badams_local/devel/compliance-checker/cchecker.py", line 41, in main
    args.format)
  File "/home/badams_local/devel/compliance-checker/compliance_checker/runner.py", line 47, in run_checker
    groups = cls.html_output(cs, score_groups, output_filename, ds_loc, limit)
  File "/home/badams_local/devel/compliance-checker/compliance_checker/runner.py", line 93, in html_output
    cs.html_output(checker, groups, f, ds_loc, limit)
  File "/home/badams_local/devel/compliance-checker/compliance_checker/suite.py", line 264, in html_output
    file_object.write(str(buf))
TypeError: unicode argument expected, got 'str'
```

```
compliance-checker --test cf -f html -- compliance_checker/tests/data/2dim-grid.nc   
Running Compliance Checker on the dataset from: compliance_checker/tests/data/2dim-grid.nc
Traceback (most recent call last):
  File "/home/badams_local/.virtualenvs/cchecker/bin/compliance-checker", line 9, in <module>
    load_entry_point('compliance-checker==2.1.0', 'console_scripts', 'compliance-checker')()
  File "/home/badams_local/devel/compliance-checker/cchecker.py", line 41, in main
    args.format)
  File "/home/badams_local/devel/compliance-checker/compliance_checker/runner.py", line 47, in run_checker
    groups = cls.html_output(cs, score_groups, output_filename, ds_loc, limit)
  File "/home/badams_local/devel/compliance-checker/compliance_checker/runner.py", line 93, in html_output
    cs.html_output(checker, groups, f, ds_loc, limit)
  File "/home/badams_local/devel/compliance-checker/compliance_checker/suite.py", line 263, in html_output
    buf = template.render(**template_vars)
  File "/home/badams_local/.virtualenvs/cchecker/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/badams_local/.virtualenvs/cchecker/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/badams_local/devel/compliance-checker/compliance_checker/data/templates/ccheck.html.j2", line 234, in top-level template code
    <td>{{score['value'][0]}}/{{score['value'][1]}}</td>
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 0: ordinal not in range(128)
```

**Comment by benjwadams** [comment]
Have a fix in the works, trying to create a few integration tests.

**Comment by benjwadams** [comment]
Fixed by #257.  Closing.
